frontend/src/bundles/common/components/base-layout/helpers/get-timestamp-n-days-ago.ts

```diff
@@ -0,0 +1,9 @@
+interface Properties {
+    days: number;
+}
+
+const getTimestampNDaysAgo = ({ days }: Properties): number => {
+    return Date.now() - days * 24 * 60 * 60 * 1000;
+};
+
+export { getTimestampNDaysAgo };
```

Comment:
I think since we already have date-fns maybe we could use differenceInDays?

Reply:
Yes, I fixed it. Thank you)

Reply:
thanks, fixed.